code/modules/reagents/Chemistry-Reagents/Toxin-Reagents.dm

```diff
@@ -38,7 +38,8 @@
 		if(prob(98))
 			randmutb(M)
 		else
-			randmutg(M)
+			var/datum/mutation/human/HM = pick((good_mutations) - mutations_list[HULK])
+			HM.force_give(M)
```

Comment:
I'd just override the randmutg() proc with a second variable that defaults to 0. Then force it to be randmutg(M, 1) and if the second argument is true, it doesn't give certain powers (for example, hulk).